app/templates/02_programming_overview.md

```diff
@@ -1,13 +1,17 @@
 <<[prev]({{int_running_code}}) [index]({{int_index}}) [next]({{int_first_steps}})>>
 # Programming is Simple
-Programming is both a broad and a deep field, however, getting your foot in the door shouldn't be intimidating.
+Programming is both a broad and a deep field; however, getting your foot in the door shouldn't be intimidating.
+
 Programming is the act of telling a computer how to do things.
-Any game that you play, computer tool that you use; All of these things that are and that could be require code somewhere in the toolchain.
-With programming, you take data in, manipulate it, and you put data back out. 
-In these 3 non-mandatory steps lie the keys to harnessing the power of a computer.
-These inputs can be key presses, text, or arbitrary data.
+
+Any game that you play, computer tool that you use, app that you run - all programs that are and that could be require code somewhere in the toolchain.
```

Comment:
remove 'all programs that are and that could be'
'require code somewhere along the line'